src/main/scala/nl/knaw/dans/easy/pid/package.scala

```diff
@@ -15,11 +15,24 @@
  */
 package nl.knaw.dans.easy
 
+import scala.util.{ Failure, Success, Try }
+
 package object pid {
 
   sealed abstract class PidType(val name: String)
   case object DOI extends PidType("doi")
   case object URN extends PidType("urn")
 
   case class RanOutOfSeeds(pidType: PidType) extends Exception(s"No more ${ pidType.name } seeds available.")
+
+  // TODO copied from easy-bag-store
+  implicit class TryExtensions2[T](val t: Try[T]) extends AnyVal {
+    // TODO candidate for dans-scala-lib
+    def unsafeGetOrThrow: T = {
```

Comment:
extension method for throwing an exception when the `Try` is a `Failure`. Note that the behavior is isomorphic to `get`, but this method name declares more the intention that the user must be aware of the fact that it will uncontrollably throw the exception.